src/guide/1/essentials/installing-kuzzle/index.md

```diff
@@ -0,0 +1,378 @@
+---
+layout: full.html.hbs
+algolia: true
+title: Install Kuzzle
+order: 1
+---
+
+# Installing Kuzzle
+
+In this section we'll describe different ways of installing Kuzzle.
+
+---
+
+## Automated script
+
+The easiest way to install Kuzzle is by using our installation script:
+
+```bash
+bash -c "$(curl http://get.kuzzle.io)"
+```
+
+---
+
+## Docker
+
+Before launching Kuzzle using Docker containers, ensure that your system meets the following requirements:
+
+- **64-bit environment**
```

Comment:
I think there should be a memory requirement here, @benoitvidis has added the corresponding checks to the installer script.